Modelica/Media/package.mo

```diff
@@ -2149,6 +2149,8 @@ package Examples
   model SimpleLiquidWater "Example for Water.SimpleLiquidWater medium model"
     extends Modelica.Icons.Example;
 
+    constant SI.PressureRate pressureRate = 1e4;
```

Comment:
```suggestion
    constant SI.PressureRate pressureRate = 1e5/10;
```
Yes, I know it is the same, but 1e5 is such a well-known pressure that I believe it to be clearer.